src/views/RegisterForm/RegisterForm.jsx

```diff
@@ -0,0 +1,342 @@
+import React, { useState } from "react";
```

Comment:
seems like most of the stuff I commented on the last PR is still here. Can you go back and have a look